source/databricks/calculation_engine/package/optimise_job/optimisation.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from delta.tables import DeltaTable
+from package.infrastructure import initialize_spark
+from pyspark.sql import SparkSession
+from package.infrastructure.paths import (
+    WholesaleResultsInternalDatabase,
+    WholesaleBasisDataInternalDatabase,
+    WholesaleInternalDatabase,
+)
+from package.common.logger import Logger
+import package.infrastructure.environment_variables as env_vars
+
+
+def optimise_tables() -> None:
+    logger = Logger(__name__)
+
+    spark = initialize_spark()
+    catalog_name = env_vars.get_catalog_name()
+
+    schema_table_dicts = {
+        f"{catalog_name}.{WholesaleResultsInternalDatabase.DATABASE_NAME}": WholesaleResultsInternalDatabase.TABLE_NAMES,
+        f"{catalog_name}.{WholesaleBasisDataInternalDatabase.DATABASE_NAME}": WholesaleBasisDataInternalDatabase.TABLE_NAMES,
+        f"{catalog_name}.{WholesaleInternalDatabase.DATABASE_NAME}": WholesaleInternalDatabase.TABLE_NAMES,
+    }
+
+    total_tables = sum(len(table_names) for table_names in schema_table_dicts.values())
+    logger.info(f"Total number of tables to optimise: {total_tables}")
+    for schema_name, table_names in schema_table_dicts.items():
+        logger.info(f"Running optimise for tables in schema: {schema_name}")
+        for table_name in table_names:
+            optimise_table(spark, schema_name, table_name, logger)
+
+
+def optimise_table(
```

Comment:
Use [PEP 8](https://peps.python.org/pep-0008/#method-names-and-instance-variables) standard for non-public symbols:
```suggestion
def _optimise_table(
```

Reply:
I can make it non-public, but it's the method being exposed and used in the test. Would that break your convention to test a non-public method.